menu.rb

```diff
@@ -0,0 +1,28 @@
+# creates a blank line
+puts " "
+
+# prints to separate lines
+puts "Welcome to Chance's Diner!"
+puts "Tonight's specials are: "
+
+# creates dash lined separation between welcome message and menu
+puts "--" * 15
+
+# arrays from which menu items are pulled
+descriptive = ["Buttery", "Watery", "Spicy", "Crispy", "Moist", "Chunky", "Crusty", "Mushy", "Soggy", "Tongue-scorching", "Mind-blowing"]
+style = ["Brazilian", "Chinese", "Mexican", "Cajun", "Bacon-wrapped", "Cream Filled", "Deep Fried", "Cheese Crusted", "Sausage Stuffed", "Five Spice", "Beer Mop"]
+main = ["Lamb Burger", "Chicken Feet", "Veal Steak", "Catfish", "Ravioli", "Octopus", "Sea Urchin", "Chicken Skin", "Gizzard", "Geoduck", "Beef Tongue"]
+
+# counts how many items are in the array
+count = descriptive.length
```

Comment:
To account for that type of edge case, what are your thoughts on creating a comparative if-statement to measure the length of each array and pop off any extra elements from the longer arrays? Any better ways I might approach this?